source/content/guides/filesystem/03-large-files.md

```diff
@@ -64,17 +69,15 @@ You can also refer to our documentation for [Drupal](/drupal-s3) and [WordPress]
 
 ### Upload Speed
 
-Uploading large files over a slow local internet connection can cause the process to hit our [Connection Timeout](/timeouts/#timeouts-that-are-not-configurable) of 59 seconds. For example, a 10MiB file uploaded on a 2Mbps connection may take too long and fail. You can use an upload time calculator like [this one](https://downloadtimecalculator.com/Upload-Time-Calculator.html) to help determine if your local internet connection is impeding file uploads to Pantheon.
+If you're transferring large files over a slow internet connection, the upload might exceed our 59-second [Connection Timeout](/timeouts/#timeouts-that-are-not-configurable). Use an [upload time calculator](https://downloadtimecalculator.com/Upload-Time-Calculator.html) to ascertain if your connection speed might prevent uploads from completing in time.
 
 ### Media and Email
 
 Refer to the [Media and Email](/guides/platform-considerations/media-email-support) section of the [Platform Considerations](/guides/platform-considerations) guide for more information on support for large media and email files.
 
 ## Large (>100GiB) File Backups
 
-Large backups take longer, use more resources, and have a higher likelihood of failing. A 100GiB compressed tarball is not a convenient solution. Sites with footprints over 200GiB or two million files cannot be backed up for this reason (although code and database are backed up as normal).
-
-Despite the lack of backups, file content is highly durable and stored on multiple servers.
+Large backups take longer, use more resources, and have a higher likelihood of failing. Sites exceeding 200GiB or containing over two million files can't be backed up (though code and databases are backed up as normal). Even without backups, file content is reliably stored across multiple servers.
```

Comment:
```suggestion
Large backups take longer, use more resources, and have a higher likelihood of failing.
Filesystems exceeding 200 GiB or containing over two million files cannot be backed up (though code and databases are backed up as normal).
Even without filesystem backups, file content is protected by redundant storage.
```
"multiple servers" feels archaic in a cloud environment - how about using "redundant storage" instead?